.nvmrc

```diff
@@ -0,0 +1 @@
+6.11.2
```

Comment:
Yeah! I wasn't sure of out official stance on Node.js versioning so I just went with a safe version (the current LTS)